tests/test_driver.py

```diff
@@ -24,36 +24,34 @@
     }
 ]
 
-category_example = {
-    708: {
+category_example = [
```

Comment:
Does this change mean this PR is a breaking change?

Reply:
No.  All we pulled the categories for previously was to get the budget category and the parent category for transactions.  The code change involved in this PR accounts for the fact that the data returned in the new endpoint is in a different structure than the original endpoint.